examples/example-server-lib/floating_window_manager.cpp

```diff
@@ -63,6 +63,17 @@ FloatingWindowManagerPolicy::FloatingWindowManagerPolicy(
     active_workspace = key_to_workspace[KEY_F1];
 }
 
+FloatingWindowManagerPolicy::FloatingWindowManagerPolicy(
+    WindowManagerTools const& tools,
+    std::shared_ptr<SplashSession> const& spinner,
+    miral::InternalClientLauncher const& launcher,
+    std::function<void()>& shutdown_hook,
+    bool focus_stealing_prevention) :
+    FloatingWindowManagerPolicy(tools, spinner, launcher, shutdown_hook)
+{
+    this->focus_stealing_prevention = focus_stealing_prevention;
+}
```

Comment:
You could avoid updating `this->focus_stealing_prevention` and use initialisers (and `const`) by forwarding from the old constructor to the new.